src/WebJobs.Extensions.NotificationHubs/Config/NotificationHubsConfiguration.cs

```diff
@@ -41,10 +44,57 @@ public void Initialize(ExtensionConfigContext context)
             {
                 throw new ArgumentNullException("context");
             }
+
+            INameResolver nameResolver = context.Config.NameResolver;
+            IExtensionRegistry extensions = context.Config.GetService<IExtensionRegistry>();
             var converterManager = context.Config.GetService<IConverterManager>();
             converterManager = converterManager.AddNotificationHubConverters();
```

Comment:
@MikeStall src/WebJobs.Extensions.NotificationHubs/Config/Converter.cs handles conversions.  Having it in a separate class makes it easier for unit testing